### PR TITLE
[tests-only] [CI] fix the nightly builds pipelines

### PR DESCRIPTION
**Pull request by dpakach**
Fix the cron build cases in drone.starlark.

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -129,7 +129,7 @@ def main(ctx):
     notify(ctx),
   ]
 
-  if ctx.build.cron != "":
+  if ctx.build.cron:
```
Comment (phil-davis) [inline .drone.star:132]: What happens in drone - I thought that this was a string?
Can you write some comments in the PR and switch it from draft to "real".

**Review by phil-davis — APPROVED** [review]

**Review by wkloucek — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Can we put a link in this PR to somewhere in drone source code where the known valid string values of `ctx.build.event` are visible (source code, or documentation). Otherwise it is all magic.

**Comment by wkloucek** [comment]
see https://docs.drone.io/pipeline/triggers/ -> By Event

```
trigger:
  event:
  - cron
  - custom
  - push
  - pull_request
  - tag
  - promote
  - rollback
```

**Comment by phil-davis** [comment]
Rebased - let's see if we can get green CI now, after other `.drone.star` changes were also merged in the last hours.

**Pull request merged**